.platform.app.yaml

```diff
@@ -91,6 +93,10 @@ relationships:
     cvocni_solr: 'solr:cvocni'
     nijac: 'db:nijac'
     nijac_solr: 'solr:nijac'
+    crc: 'db:crc'
+    crc_solr: 'solr:crc'
+    sentencereview: 'db:sentencereview'
+    sentencereview_solr: 'solr:sentencereview'
     src: 'db:src'
     src_solr: 'solr:src'
```

Comment:
Looks like you are adding crc back in and not deleting the old src? This is common throughout most of these files